index.html

```diff
@@ -32,6 +32,8 @@
           </div>
           <div class="h-[60px] w-[20px] rounded-r-full bg-white"></div>
         </div>
+        <div id="quote-container"> <!-- Enter Quote Here --> </div>
```

Comment:
We only need one container for now, give the container an id of something that describes both jokes and quotes, something like 'text-container'.